### PR TITLE
Restore v1alpha1 DevWorkspace CRD

### PR DESCRIPTION
**Pull request by amisevsk**
### What does this PR do?:
This commit effectively reverts https://github.com/devfile/api/pull/639, restoring the `v1alpha1` version of the DevWorkspace to the devfile API. This is required as, after we merged the PR above, we discovered that the OpenShift Console still depends on the v1alpha1 apiVersion.

### Which issue(s) this PR fixes:
Closes https://github.com/devfile/api/issues/721

### PR acceptance criteria:
- [x] Unit/Functional tests

- [ ] [QE Integration test](https://github.com/devfile/integration-tests) 
    - I believe no changes are required here

- [ ] Documentation 
    - Documenation is unchanged

- [ ] Client Impact
    - No breaking changes

### How to test changes / Special notes to the reviewer:
Test that v1alpha1 API is restored to CRDs. The Devfile schema should be unchanged.

### CODE REVIEW

File: crds/workspace.devfile.io_devworkspaces.v1beta1.yaml [hunk 1]
```diff
@@ -31,3936 +18,8086 @@ spec:
...
+                                              - internal
+                                              - none
+                                              type: string
+                                            name:
+                                              type: string
+                                            path:
+                                              description: Path of the endpoint URL
+                                              type: string
+                                            protocol:
+                                              description: "Descr
```
Comment (yangcao77) [inline crds/workspace.devfile.io_devworkspaces.v1beta1.yaml]: those fields added in v1alpha2 should not be included in v1alpha1? similar as other files.

File: pkg/apis/workspaces/v1alpha1/devworkspace_conversion.go [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package v1alpha1
+
+import (
+	"github.com/devfile/api/v2/pkg/apis/workspaces/v1alpha2"
+	"sigs.k8s.io/controller-runtime/pkg/conversion"
```
Comment (yangcao77) [inline pkg/apis/workspaces/v1alpha1/devworkspace_conversion.go:5]: this go module is missing in vendor

**Review by yangcao77 — CHANGES_REQUESTED** [review]
Since it's a revert of #639 , I don't see any difference compare with files before changes in #639 . Just need to add one missing vendor.

**Comment by yangcao77** [comment]
changes look good. just need to pass the ci, and I will add the label.

**Comment by amisevsk** [comment]
Running build.sh makes no changes locally, and trying `./docker-run.sh ./build.sh` gives 
```
check ./build.sh
Building image quay.io/devfile/kubernetes-api-build-prerequisites:latest...[OK]
Running ./build.sh
bash: line 1: ./build.sh: Permission denied
Fail to run the script
```
:thinking:

**Review by yangcao77 — APPROVED** [review]

**Pull request merged**